plugins/reporters/cyclonedx/src/main/kotlin/CycloneDxReporter.kt

```diff
@@ -356,17 +363,16 @@ class CycloneDxReporter : Reporter {
         schemaVersion: Version,
         outputDir: File,
         outputName: String,
-        requestedOutputFileFormats: Set<FileFormat>
+        validFileExtensions: List<String>
```

Comment:
Propose to name this just `outputFileExtensions` for consistency with other parameters.